comm/src/main/scala/coop/rchain/p2p/p2p.scala

```diff
@@ -61,22 +61,33 @@ object Network extends ProtocolDispatcher[java.net.SocketAddress] {
   def connect[F[_]: Capture: Monad: Log: Time: Metrics: Communication: Encryption](peer: PeerNode)(
       implicit
       keysStore: KeysStore[F],
```

Comment:
Just out of curiosity: Why is it not possible to pull up `KeysStore` into the type signature above?